edc-extensions/edr/edr-callback/src/test/java/org/eclipse/tractusx/edc/callback/ContractNegotiationCallbackTest.java

```diff
@@ -127,9 +126,9 @@ private static class EventInstances implements ArgumentsProvider {
         public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
             return Stream.of(
                     baseBuilder(ContractNegotiationAccepted.Builder.newInstance()).build(),
-                    baseBuilder(ContractNegotiationConfirmed.Builder.newInstance()).build(),
-                    baseBuilder(ContractNegotiationDeclined.Builder.newInstance()).build(),
-                    baseBuilder(ContractNegotiationFailed.Builder.newInstance()).build(),
+                    baseBuilder(ContractNegotiationAgreed.Builder.newInstance()).build(),
+                    // baseBuilder(ContractNegotiationFinalized.Builder.newInstance()).build(),
```

Comment:
probably this should be removed or supported in the test

Reply:
moved it into a separate test method, because the callback behaves differently on a `ContractNegotiationFinalized` event.